src/website2/src/components/ui/MemberCard.tsx

```
@@ -174,27 +210,29 @@ const MemberCard: React.FC<MemberCardProps> = ({
+                        <p key={idx} className="mb-2">
+                          {desc.description}
+                        </p>
+                      ),
+                    )
                   : null}
 
-                {member.bio && (
+                {(detailed as any).bio && (
                   <div
                     className="mb-2"
                     dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)